framework/include/userobject/DomainUserObject.h

```diff
@@ -101,10 +112,14 @@ class DomainUserObject : public UserObject,
   /**
    * Routes through to \p Coupleable::getFieldVar, but also inserts the return variable into a set
    * of field variables to check on interface-connected blocks, as opposed to our blocks, when
-   * performing our block-restriction integrity check
+   * performing our block-restriction integrity check.
+   * The argument \p interfaces is optional specifying on what interfaces the variable are expected
+   * to be defined. Default value means that the variable should be defined on all interfaces.
```

Comment:
let s be more precise. Is this really turning it on for ALL internal sidesets?
or just near the domain

Reply:
Replaced with:
```
   * The argument \p interfaces is optional specifying on what interfaces the variable is expected
   * to be available, i.e. the field variable is defined over elements out side of the domain but
   * connecting the subdomain with the interface. Default value means that the variable should be
   * available on all interfaces.
   * Note: a field variable on interface is not required to be defined on the subdomain of this
   *       domain user object.
```